t1t2_mwpm/circuits/rep_code.py

```diff
@@ -0,0 +1,213 @@
+"""Module containing functions for generating circuits for a repetition code memory experiment."""
```

Comment:
Yes, but I believe that the circuit that they are running is not exactly the one you wrote in the file. Mainly because they cannot do all the CZs at the same time due to qubit frequency collisions. They follow the same CZ dance as the surface code.

Reply:
OK, I see you're taking care of that in the repetition code experiment repo. For now I've deleted it.